lib/mix_test_interactive/watcher.ex

```diff
@@ -11,6 +11,8 @@ defmodule MixTestInteractive.Watcher do
 
   require Logger
 
+  @ignore_events MapSet.new([:isdir, :closed, :attribute, :undefined])
```

Comment:
This list looks reasonable for Linux and Windows-based systems, but not for MacOS systems. The FSMac backend supports a [significantly larger set of possible events](https://github.com/falood/file_system/blob/v1.0.0/lib/file_system/backends/fs_mac.ex#L73-L95).

I wonder if it would be better to instead list the events that we care about and then reverse the logic below. For the existing backends, that would likely be: `[:created, :deleted, :modified, :removed, :renamed]`.